react/features/welcome/functions.js

```diff
@@ -46,3 +51,19 @@ export function isWelcomePageUserEnabled(stateful: Function | Object) {
             ? true
             : toState(stateful)['features/base/config'].enableWelcomePage);
 }
+
+/**
+ * Render header arrow back button for navigation.
+ *
+ * @param {Function} onPress - Callback for when the button is pressed
+ * function.
+ * @returns {ReactElement}
+ */
+export function renderArrowBackButton(onPress: Function) {
+    return (
+        <HeaderNavigationButton
+            /* eslint-disable-next-line react/jsx-no-bind */
```

Comment:
Is this needed?

Reply:
Yes we need it for replacing the burger icon on Settings, Terms, Privacy and Help screens.

Reply:
I mean the eslint-disable :-P

Reply:
Removed